packages/front-end/components/GuidedGetStarted/GuidedGetStarted.tsx

```diff
@@ -317,7 +361,7 @@ export default function GuidedGetStarted({
     if (initialStep >= 0) {
       return initialStep;
     } else {
-      return 0;
+      return steps.length - 1;
```

Comment:
Will this behave correctly when the user hasn't done any steps yet?

Reply:
Hmm... it should - if a user hasn't done any steps yet, the `steps.findIndex` function just above this should stop on the first step of the steps array, and then this check will just return `initialStep`.